docker/docker-compose.yml

```diff
@@ -114,16 +89,17 @@ services:
       CONSEIL_LORRE_DEPTH: "newest"
       CONSEIL_LORRE_HEAD_HASH: "head"
 
-      CONSEIL_XTZ_NETWORK: "granadanet"
+      CONSEIL_XTZ_NETWORK: "jakartanet"
       CONSEIL_XTZ_ENABLED: "true"
-      CONSEIL_XTZ_NODE_HOSTNAME: "tezos-granada.cryptonomic-infra.tech"
-      CONSEIL_XTZ_NODE_PORT: 443
+      CONSEIL_XTZ_NODE_PROTOCOL: "http"
+      CONSEIL_XTZ_NODE_HOSTNAME: "172.17.0.1"
```

Comment:
This appears to be an IP in a range that docker likes to use. Note that there is no guarantee that docker will assign this IP to a tezos node always. Perhaps it might be best to leave upto the user to configure.